### ISSUE TITLE
Piped content boundaries are not correct

### ISSUE THREAD

**Issue by sattellite** [charmbracelet/freeze#50]
**Describe the bug**
The text extends beyond the borders of the image.

**To Reproduce**

<details>

<summary>config.go</summary>

```go
package config

import (
	"fmt"
	"os"
	"reflect"
	"strconv"
	"strings"
)

type Config struct {
	Telegram struct {
		Token   string `env:"TG_TOKEN"`
		ChatID  string `env:"TG_CHAT"`
		OwnerID string `env:"TG_ADMIN"`
	}

	Database struct {
		DSN string `env:"DB_DSN"`
	}

	Log struct {
		Level string `env:"LOG_LEVEL"`
	}

	Debug bool
}

func Load() (*Config, error) {
	var c Config
	err := parseStruct(&c, "")
	if err != nil {
		return nil, err
	}
	return &c, nil
}
```
</details>

`cat config.go | freeze --language go --theme murphy`

**Expected behavior**
I expect the image to be a little wider so that the data inside the tags will fit completely into the image. And also the colored background of the symbols will be in correct place.

**Screenshots**
<details>
<summary>Resulted image</summary>

![screenshot](https://github.com/charmbracelet/freeze/assets/322910/1d4d01bf-4846-44e6-b743-518d9ad96fe1)

</details>

**Desktop (please complete the following information):**
 - macOS 14.4
 - iTerm 3.5.0beta17

**Comment by maaslalani** [comment]
Hey @sattellite, thanks for the issue! I am unable to reproduce. When I run the following:

```
cat config.go | freeze --language go --theme murphy
```

I get the following:

![freeze](https://github.com/charmbracelet/freeze/assets/42545625/3a2b6bf9-afe3-4656-8f04-55f62cf6d42c)

NB: This conversion is using `libsvg` (rsvg-convert) which is installed on my computer. Let me try again with the normal `resvg-go`.

**Comment by sattellite** [comment]
Try to use `config.go` from my message, please. I can reproduce that behaviour on intel mac and m2 mac. First message from m2 mac.

freeze 0.1.4 installed with `go install github.com/charmbracelet/freeze@latest` command on both computers.

<details>
<summary>screenshot from intel mac</summary>

![freeze](https://github.com/charmbracelet/freeze/assets/322910/213472ab-efd4-47e5-9156-14a566085861)

</details>

**Comment by maaslalani** [comment]
Ah you're absolutely correct @sattellite, I am able to reproduce with your code. Thank you, I will fix this issue!

**Comment by maaslalani** [comment]
@sattellite The issue was tabs in the code were causing miscalculations in the width. This is now fixed on main! Thank you for the report and awesome reproduction instructions!

**Comment by maaslalani** [comment]
![freeze](https://github.com/charmbracelet/freeze/assets/42545625/cf17a234-b9f3-4073-8ef2-58f0ac7cd4bd)
